models/articles.model.js

```diff
@@ -0,0 +1,14 @@
+const db = require("../db/connection.js");
+
+function selectArticleById(article_id) {
+  return db
+    .query("SELECT * FROM articles WHERE article_id = $1;", [article_id])
+    .then((result) => {
+      if (result.rows.length === 0) {
+        return Promise.reject({ statusCode: 404, msg: "team does not exist" });
```

Comment:
What kind of message is this in regards to articles?????

Reply:
This is for sure a lesson in copy&pasting code!! Thanks for your feedback Verity!